io.catenax.mandatory_dismantling/1.0.0/MandatoryDismantling.ttl

```diff
@@ -0,0 +1,994 @@
+
+:dashboard a samm:Property ;
+   samm:preferredName "Dashboard"@en ;
+   samm:description "A property of the model, named  dashboard."@en ;
+   samm:characteristic :DashboardCharacteristic .
+
+:infotainmentSystem a samm:Property ;
+   samm:preferredName "Infotainment System"@en ;
+   samm:description "A property of the model, named  infotainment system."@en ;
+   samm:characteristic :InfotainmentSystemCharacteristic .
+
+:headlight a samm:Property ;
+   samm:preferredName "Headlight"@en ;
+   samm:description "A property of the model, named  headlight."@en ;
+   samm:characteristic :HeadlightCharacteristic .
+
+:wireHarness a samm:Property ;
+   samm:preferredName "Wire Harness"@en ;
+   samm:description "A property of the model, named wire harness."@en ;
+   samm:characteristic :WireHarnessCharacteristic .
+
+:bumper a samm:Property ;
+   samm:preferredName "Bumper"@en ;
+   samm:description "A property of the model, named bumper."@en ;
+   samm:characteristic :BumperCharacteristic .
+
+:fluidContainer a samm:Property ;
+   samm:preferredName "Fluid Container"@en ;
+   samm:description "A property of the model, named fluid container."@en ;
+   samm:characteristic :FluidContainerCharacteristic .
+
+:heatExchanger a samm:Property ;
+   samm:preferredName "Heat Exchanger"@en ;
+   samm:description "A property of the model, named heat exchanger."@en ;
+   samm:characteristic :HeatExchangerCharacteristic .
+
+:metalComponent a samm:Property ;
+   samm:preferredName "Metal Component"@en ;
+   samm:description "A property of the model, named metal component."@en ;
+   samm:characteristic :MetalComponentCharacteristic .
+
+:plasticComponent a samm:Property ;
+   samm:preferredName "Plastic Component"@en ;
+   samm:description "A property of the model, named plastic component."@en ;
+   samm:characteristic :PlasticComponentCharacteristic .
+
+:inverter a samm:Property ;
+   samm:preferredName "Inverter"@en ;
+   samm:description "A property of the model, named inverter."@en ;
+   samm:characteristic :InverterCharacteristic .
+
+:printedCircuitBoard a samm:Property ;
+   samm:preferredName "Printed Circuit Board"@en ;
+   samm:description "A property of the model, named printed circuit board."@en ;
+   samm:characteristic :PrintedCircuitBoardCharacteristics .
+
+:pvPanel a samm:Property ;
+   samm:preferredName "PV Panel"@en ;
+   samm:description "A property of the model, named PV panel."@en ;
+   samm:characteristic :PVPanelCharacteristic .
+
+:transmissionComponent a samm:Property ;
+   samm:preferredName "Transmission Component"@en ;
+   samm:description "A property of the model, named transmission component."@en ;
+   samm:characteristic :TransmissionComponentCharacteristic .
+
+:QuantityCharacteristic a samm-c:Quantifiable ;
+   samm:preferredName "Quantity Characteristic"@en ;
+   samm:description "Describes a Property which contains a number."@en ;
+   samm:dataType xsd:integer .
+
+:WeightCharacteristic a samm-c:Measurement ;
```

Comment:
I would suggest you look into the shared quantity model. It describes for example weight values, (where you can also choose the unit)